sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobSetTierHeaders.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+/**
+ * Defines headers for SetTier operation.
+ */
+@JacksonXmlRootElement(localName = "Blob-SetTier-Headers")
+public class BlobSetTierHeaders {
+    /*
```

Comment:
If we include property descriptions in our Javadoc we don't need to mention them again in the getters and setters. Not sure what the convention is in the Java SDKs but I guess code would look cleaner this way.
```suggestion
    /**
```

Reply:
These were all simply copy pasted from the Java SDK. In the Java SDK we generally like to keep these "generated" types hidden from customers. I think that might be the way we might choose to go here as well. I'd rather just leave it to be as close to existing Java generated code as possible

Reply:
Given that we now have an AutoRest implementation for Android, we might want to re-generate these classes and revisit this at some point to see if we need any convenience added on top of the models.

Reply:
I can make that a work item since its likely that this will affect all the other APIs